extensions/amp-image-slider/validator-amp-image-slider.protoascii

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-image-slider
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-image-slider"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-image-slider>
+  html_format: AMP
+  html_format: EXPERIMENTAL
+  tag_name: "AMP-IMAGE-SLIDER"
+  requires_extension: "amp-image-slider"
+  attr_lists: "extended-amp-global"
+  spec_url: "https://www.ampproject.org/docs/reference/components/amp-image-slider"
+  attrs: {
```

Comment:
What about `initial-percent` and `step-size`?

Reply:
Sync-ed offline and we'll add in all the relevant validation for P1 features after they are finalized.